programmers/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,10 @@
+function solution(lottos, win_nums) {
+    const ranks = [6, 6, 5, 4, 3, 2, 1];
+    
+    const check = lottos.filter(el => win_nums.includes(el)).length;
+    const random = lottos.filter(zero => !zero).length;
+    
+    const max = check + random;
+    
+    return [ranks[max], ranks[check]];
```

Comment:
```suggestion  
    return [ranks[check + random], ranks[check]];
```
두 번 이상 사용하지  않거나, 간단한 처리만 거친 거라면 변수 할당을 지양하는 것이 좋습니다.